server/src/middleware/auth.js

```diff
@@ -4,17 +4,17 @@ const keys = require("../config/keys");
 
 const auth = async (req, res, next) => {
   // Get token from the header
-  const token = req.header("x-auth-token");
+  var token = req.header("Authorization");
```

Comment:
Why have you changed this file?

Reply:
Actually, for some reason postman is not detecting the token if it is not stored in Authorization.

Reply:
![image](https://user-images.githubusercontent.com/44175278/99557409-aa39d380-29e8-11eb-8b9f-acb56e4088c4.png)

No no, it works absolutely fine.